packages/graphql-serve/src/commands/serve.ts

```diff
@@ -25,6 +25,17 @@ export const builder = (args: yargs.Argv): void => {
     type: 'boolean',
     alias: 'ds'
   })
+
+  args.option('conflict', {
+    describe:"Specify a conflict resolution strategy with --datasync",
+    type: 'string',
+    choices: ["client", "server"]
```

Comment:
In my opinion having just "client" and "server" is not obvious enough as to what they do. If the choices were `serverSideWins` or `clientSideWins` it would become much clearer, and self-documenting.